src/chatdlg.cpp

```diff
@@ -132,7 +132,7 @@ void CChatDlg::OnAnchorClicked ( const QUrl& Url )
     {
         if ( QMessageBox::question ( this,
                                      APP_NAME,
-                                     tr ( "Do you want to open the link" ) + " <b>" + Url.toString() + "</b> " + tr ( "in an external browser?" ),
+                                     tr ( "Do you want to open the link %1 in an external browser?" ).arg ( " <b>" + Url.toString() + "</b> " ),
```

Comment:
```suggestion
                                     tr ( "Do you want to open the link « %1 » in an external browser?" ).arg ( " <b>" + Url.toString() + "</b> " ),
```

or

```suggestion
                                     tr ( "Do you want to open the link '%1' in an external browser?" ).arg ( " <b>" + Url.toString() + "</b> " ),
```

if that works.